pages/apim/3.x/installation-guide/configuration/repositories/installation-guide-repositories-jdbc.adoc

```diff
@@ -106,3 +106,28 @@ NOTE: Before running any scripts, you need to create a dump of your existing dat
 . Rename your tables using format `prefix_tablename`.
 . Rename your indexes using format `idx_prefix_indexname`.
 . Rename your primary keys using format `pk_prefix_pkname`.
+
+[[database-enforcing-use-of-primary-key-on-all-tables]]
+== Database enforcing use of primary key on all tables
+
+Some databases have an option to enforce the use of a primary key on all tables, for instance, this is the case for MySQL 8.0.13+ with `sql_require_primary_key`.
+
+If you are using a database with such option activated, during the *installation of APIM*, you will need to:
+
+ - disable this option
+ - start APIM Management API, to allow the database migration tool, Liquibase, to create the APIM tables and add the primary keys
+ - manually add a primary key on the `DATABASECHANGELOG` table following https://docs.liquibase.com/concepts/tracking-tables/databasechangelog-table.html[Liquibase's recommendation]: `ALTER TABLE DATABASECHANGELOG ADD PRIMARY KEY (ID, AUTHOR, FILENAME);`
+ - manually add a primary key on the `DATABASECHANGELOGLOCK` table following https://docs.liquibase.com/concepts/tracking-tables/databasechangeloglock-table.html[Liquibase's recommendation]: `ALTER TABLE DATABASECHANGELOGLOCK ADD PRIMARY KEY (ID);`
```

Comment:
I just removed this part of the doc as it's already done in [this changelog](https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v3_15_20/schema.yml#L27) now